docs/tutorials/trajectories.rst

```diff
@@ -0,0 +1,67 @@
+==========================
+Handle Expert Trajectories
+==========================
+
+For imitation learning we need expert trajectories.
```

Comment:
I think for e.g. preference comparison non-expert trajectories are stored with this same data structure right? And adversarial methods use this for the non-expert policy to train/evaluate against the discriminator?

Reply:
Good point! I will adjust the wording to reflect this.